UCR/rdf-star-ucr.html

```diff
@@ -287,6 +287,76 @@ <h4>annotate commit deltas</h4>
           so that a triple can be searched for across commit history in SPARQL*.
         </p>
       </section>
+
+      <section id="rdfsSupport">
+        <h4>SPARQL*/RDF* and RDFS support</h4>
+        <p><a href="https://github.com/w3c/rdf-star/issues/80#issuecomment-777292682">Submitted Use Case</a></p>
+        <p>
+          As a user, I want to express facts as triple statements `{:s :p :o}` within a
+          triplestore.  Later on I realise I want to add facts about this fact.  However
+          the {:s :p :o} triple has already been asserted in the triplestore, so I need to
+          add a reification.
+        </p>
+        <pre>
+          :key rdf:subject :s ; rdf:predicate :p ; rdf:object :o .
+        </pre>
+        <p>
+          Now I can add facts about the fact:
+        </p>
+        <pre>
+          :key :uom :cm ; :accuracy ".32" .
+        </pre>
+        <p>
+          So far 'standard' reification.
+        </p>
+        <p>
+          However, as a user I want to add some RDFS 'referential' integrity to help
+          manage the use of additional predicates such a :uom and :accuracy.  Since the
+          rdfs:domain of rdf:subject, rdf:predicate, and rdf:object, it can be inferred
+          that :key is a rdf:Statement.  This then implies that the rdfs:domain of :uom
+          and :accuracy is also rdf:Statement. Thus if I continue reifying other
+          statements, any predicates associated with the reified statement must have a
+          rdfs:domain of rdf:Statement.
+        </p>
```

Comment:
```suggestion
          that :key is a rdf:Statement.  This then implies that the 
          schema:domainIncludes of :uom and :accuracy includes rdf:Statement. Thus if I 
          continue reifying other statements, any predicates associated with the reified 
          statement &mdash;
        </p>
        <ul>
          <li>may have an rdfs:domain of rdf:Statement</li>
          <li>must have a schema:domainIncludes of rdf:Statement</li>
        </ul>
```

There's more from @afs in https://github.com/w3c/rdf-star/issues/80#issuecomment-777824992 but I'm not sure how to fit it in here...

Reply:
This is as written by the contributor. The UCR document is not critique of the use cases.

Reply:
> This is as written by the contributor. The UCR document is not critique of the use cases.

No, but neither has any UCR document in any group I've yet participated in been simply a verbatim collection of submissions. Editorial processes are brought to bear along the way, and this is part of that way.

Reply:
The UC is RDFS, schema:domainIncludes is not in RDFS. 

The use of here schema:domainIncludes is debatable.